litmus/dumpRun.ml

```diff
@@ -261,7 +262,7 @@ let dump_shell names =
       end in
       let module RU = RunUtils.Make(O) in
       RU.report_machine out_chan ;
-      output_line out_chan "head -1 comp.sh" ;
+      output_line out_chan "sed '2q;d' comp.sh" ;
```

Comment:
Why change? Is `sed` more likely to be available than `head`?

Reply:
I believe the `head` command grabbed the first line of the `comp.sh` file (`gcc=...`) As this will now be the shebang, we instead want the second line, hence the move to `sed`. Not sure if I misunderstand, or if there's a better way?